src/py21cmfast/inputs.py

```diff
@@ -931,3 +918,47 @@ def t_STAR(self):
             raise ValueError("t_STAR must be above zero and less than or equal to one")
         else:
             return self._t_STAR
+
+
+class InputCrossValidationError(ValueError):
+    """Error when two parameters from different structs aren't consistent."""
```

Comment:
Is this used anywhere? Or is it currently incomplete?

Reply:
It's not used, but it is intended to be used in `validate_all_inputs` when something is wrong. At the moment, `validate_all_inputs` is just warning and setting things to appropriate values. I thought I'd leave this here for the future.